src/static/css/_pagination.css

```
@@ -0,0 +1,73 @@
+.pagination {
+    align-items: center;
+    background: var(--fl-bgColor, var(--pagination-bg));
```

Add a third fallback to `color-blue-100` here so you don't need to define a variable globally for this:

```suggestion
    background: var(--fl-bgColor, var(--pagination-bg, var(--color-blue-100));
```